ground/src/main/java/com/google/android/ground/model/submission/DateResponse.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.submission
+
+import java8.util.Optional
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
+ * A user-provided response to a date question task.
+ */
+@Serializable
+class DateResponse(val date: @Contextual Date) : Response {
```

Comment:
Should we make this a data class?

Reply:
Good idea, I'm not sure I realized that data classes handled the equals() toString() and hashCode() methods so this ended up much cleaner after that!

Reply:
awesome!